.github/workflows/ci_main.yml

```diff
@@ -0,0 +1,76 @@
+# Continuous integration and delivery of the main branch.
+# This workflow will run action below on every push to the main branch.
+
+name: CI/CD
+on:
+  push:
```

Comment:
Make this workflow dispatch 
- input would be whether its patch, minor, major for bumpversion